src/_variables.scss

```diff
@@ -73,33 +138,100 @@ $pagination-hover-color: $btn-default-color !default;
 $pagination-hover-bg: darken($btn-default-bg, 8%) !default;
 $pagination-hover-border: darken($btn-default-border, 12%) !default;
 
+// WET Custom
 $pager-border-radius: 4px !default;
 
-// Footnotes
+
+//
+// WET CUSTOM PLUGINS VARIABLES
+//
+
+//== Calendar
+//
+//##
+
+$cal-text-head-color: #fff !default;
+$cal-text-body-color: #000 !default;
+
+$cal-dark-color: #333 !default;
+$cal-medium-color: $clrDark !default;
+
+$cal-mnth-arrows-active-color: $clrMedium !default;
+
+//== Footnotes
+//
+//##
+
 $spacing: .375em !default;
 $ddRtnWidth: 3.375em !default;
 
-// Calendar
-$cal-text-head: #fff !default;
-$cal-text-body: #000 !default;
-$cal-dark: #333 !default;
-$cal-medium: #555 !default;
-$cal-active: #ccc !default;
-$cal-days-link: #111 !default;
 
-//Multimedia Player
+//== Multimedia player
+//
+//##
+
 $mm-progress-fg-color: #176ca7 !default;
 $mm-progress-bg-color: #fff !default;
+
 $mm-cc-fg-color: #fff !default;
 $mm-cc-bg-color: #000 !default;
+
 $mm-ctrl-fg-color: #fff !default;
 $mm-ctrl-bg-color: #3e3e3e !default;
 $mm-ctrl-outline-color: #4aafff !default;
 $mm-ctrl-cc-color: #4aafff !default;
+
 $mm-volume-bg-color: #aaa !default;
 $mm-volume-thumb-color: #fff !default;
 
-// Additional Viewports outside of Bootstrap definitions
-$screen-xxs-max: 479px !default;
-$screen-lg-max: 1599px !default;
-$screen-xl-min: 1600px !default;
+
+//== Tabs
+//
+//##
+
+// Tabbed Interface
+$tablist-color: #000 !default;
+
+$tablist-border-color: $clrMedium !default;
+$tablist-border-style: solid !default;
+$tablist-border-width: 1px !default;
+
+$tabpanel-border-color: $tablist-border-color !default;
+$tabpanel-border-style: solid !default;
+$tabpanel-border-width: 1px !default;
+
+$tablist-bg-color: #ebf2fc !default;
+$tablist-hover-bg-color: $clrMedium !default;
+$tablist-active-bg-color: #fff !default;
+
+$tablist-active-link-border-color: #666 !default;
+$tablist-active-link-border-style: solid !default;
+$tablist-active-link-border-width: 4px 0 0 0 !default;
+
+// Tablist at sm max && xxs max
+$tablist-screen-sm-details-border-color: $tablist-border-color !default;
+$tablist-screen-sm-details-border-style: solid !default;
+$tablist-screen-sm-details-border-width: 0 1px 1px 1px !default;
+$tablist-screen-sm-details-top-border-width: 1px !default;
```

Comment:
Well in the example above `$tablist-screen-sm-details-top-border-width` is not needed since `$tablist-screen-sm-details-border-width` can specify the width on each edge. Can't the same be done for the radius?

```
$tablist-screen-sm-details-border-radius: 5px 0 5px 0;
```